scripts/bert/bertpass_gpu.cc

```diff
@@ -30,6 +30,7 @@
 #include <functional>
 #include "mxnet/lib_api.h"
 
+#if MXNET_1_7
```

Comment:
If you want you can use the defines in `lib_api.h` to enable the appropriate code paths:
[MXNet 1.7](https://github.com/apache/incubator-mxnet/blob/v1.7.x/include/mxnet/lib_api.h#L52): `#define MX_LIBRARY_VERSION 7`
[MXNet 1.8](https://github.com/apache/incubator-mxnet/blob/1.8.0.rc1/include/mxnet/lib_api.h#L56): `#define MX_LIBRARY_VERSION 10`